### ISSUE TITLE
training error when running bash scripts/train.sh

### ISSUE THREAD

**Issue by alenaliu** [Legion56/Counting-ICCV-DSSINet#11]
![image](https://user-images.githubusercontent.com/11977412/76285240-9d55d380-62e2-11ea-8c95-c901fd694df0.png)

why the training log shows that the es_cnt of different imags are always same and model seems does not converge

**Comment by svshivapuja** [comment]
Yes I have observed the same thing. Did you get low test error after training was complete?
